app/src/main/res/drawable/ic_new_pin.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="10dp"
+    android:height="10dp"
+    android:viewportWidth="10"
+    android:viewportHeight="10">
```

Comment:
Looking at the figma design spec more closely, I think this should be `13dp` or it might be 12dp with 1dp padding. Perhaps we should check with @brampitoyo here.

<img width="601" alt="Screen Shot 2021-04-21 at 11 40 13 AM" src="https://user-images.githubusercontent.com/1190888/115581838-5fbdcb00-a296-11eb-8b50-dc76de786acb.png">

Reply:
@brampitoyo Suggested using the proton icon that has 10x10. I am fine with either one.

Reply:
Gotcha, I am fine using 10x10 pathData. I am wondering about the final size if it should be 13x13

Reply:
I will ping @brampitoyo for the final review